src/lib/y2storage/disk_analyzer.rb

```diff
@@ -507,5 +533,22 @@ def partitions_with_id(ids, log_label)
       log.info("Found #{log_label} partitions: #{result}")
       result
     end
+
+    # Name of the LVM volume group to which the partition belongs
+    #
+    # @param partition [::Storage::Partition]
+    # @return [String] nil if the partition does not hold a physical volume
+    def vg_for(partition)
+      name = partition.name
+      lvm_pv = ::Storage::LvmPv.all(devicegraph).to_a.detect { |pv| pv.blk_device.name == name }
+      return nil unless lvm_pv
+
+      begin
+        lvm_pv.lvm_vg.vg_name
+      rescue ::Storage::WrongNumberOfChildren
```

Comment:
Due to a crash in libstorage-ng, I cannot verify this would be the correct way of checking whether a PV is unassigned. @aschnell, is correct?